### ISSUE TITLE
Table, numbers indicating the order in multiple sorting

### ISSUE THREAD

**Issue by nicoabie** [ant-design/ant-design#30574]
- [X] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
When a table is being sorted by different columns, it would be nice for the user to remember the order in which the sorts were applied.

### What does the proposed API look like?
Some flag in the table component called includeSortOrder or similar which defaults to false to maintain current behavior.

**Comment by nicoabie** [comment]
@afc163 If it gets accepted maybe I can try putting a PR with some guidance.

**Comment by afc163** [comment]
There is `sorter.mutiple` for priority: https://ant.design/components/table/#components-table-demo-multiple-sorter

**Comment by nicoabie** [comment]
Yes, but i don't want to impose any priority to the user. I want her/him to define the custom order an remind them which was it.
